packages/http-client/src/ceramic-http-client.ts

```diff
@@ -73,11 +66,12 @@ export default class CeramicClient implements CeramicApi {
   private readonly _config: CeramicClientConfig
   public readonly _doctypeConstructors: Record<string, DoctypeConstructor<Doctype>>
 
-  constructor (apiHost: string = CERAMIC_HOST, config: CeramicClientConfig = {}) {
+  constructor (apiHost: string = CERAMIC_HOST, config: Partial<CeramicClientConfig> = {}) {
     this._config = { ...DEFAULT_CLIENT_CONFIG, ...config }
 
     this._apiUrl = combineURLs(apiHost, API_PATH)
-    this._docCache = new DocCache(config.docCacheLimit, this._config.cacheDocCommits)
+    // this._docCache = new LRUMap(config.docCacheLimit) Not now. We do not know what to do when document is evicted on HTTP client.
```

Comment:
Do we even still need this `_docCache` in the http client?  What is it gaining us?

EDIT: Nevermind, feel free to ignore this for now, we can revisit this in the cleanup http client project